index.html

```diff
@@ -18,16 +21,178 @@
             </div>
         </nav>
     </header>
-    <!-- END NAVBAR SECTION-->
+    <!-- End NAVBAR SECTION-->
 
-    <!-- START HEADLINE SECTION-->
+    <!-- Start HEADLINE SECTION-->
     <section class="headline">
         <div class="content">
             <h1 class="author">Thiri Htet Htet Aung</h1>
             <p class="intro">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
             <button type="button" class="collab-btn">Start collaboration</button>
         </div>
     </section>
-    <!-- END HEADLINE SECTION -->
+    <!-- End HEADLINE SECTION -->
+
+    <!-- Start WORK SECTION -->
+    <section class="works">
+        <!-- start title -->
+        <div class="title">
+            <h1>Projects</h1>
+        </div>
+        <!-- end title -->
+
+        <!-- start project card -->
+        <div class="projects">
+            <!-- start card1 -->
+            <div class="project-cards project1">
+                <div class="pjimgs">
+                    <img src="./img/pj1.svg" alt="pj1"/>
+                </div>
+                <div class="pjcontent">
+                    <p class="pjname">Project name goes here</p>
+                    <ul>
+                        <li>HTML/CSS</li>
+                        <li>Ruby on Rails</li>
+                        <li>Javasccript</li>
+                    </ul>
+                    <button class="pjbtn"><a href="#"><span>See this Project</span> <img src="./img/union.svg" alt="rightarrow"/></a></button>
```

Comment:
All occurrences of a button in each card should at least look like the design from figma. Currently your buttons do not have the rightward pointing arrow as the template. 

Your copy 👇👇

<img width="221" alt="no arrow" src="https://user-images.githubusercontent.com/84930355/205228270-54437db4-a304-4526-b5ea-412144a4b575.png">

Template design 👇👇

<img width="383" alt="with arrow" src="https://user-images.githubusercontent.com/84930355/205228354-8e6cc75d-7ebb-4d4b-b67d-208e83187802.png">


Kindly implement this according to the design and also take care of the wrapping text by reducing the font size.